docs/component/config-provider.md

```
@@ -274,14 +274,72 @@ useConfigProvider({ themeVars })
+</wd-config-provider>
+```
+
+:::
+
+::: tip 关于命令式 API
+`useToast()` / `useDialog()` 等命令式 API 是通过函数式调用使用的，**不读取** `ConfigProvider` 配置，需要在调用时直接传入 `options`（如 `useToast().show({ msg, duration })`）。这是 wot-ui 当前的设计取舍，目的是避免模块单例在 Vite 预构建 / 多实例场景下带来的同步问题。
+:::
+
+### 配置优先级
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)